### DISCUSSION TITLE
Cannot get chinese character as slug

### DISCUSSION THREAD

**Discussion by blu94** [spatie/laravel-sluggable#244] [Q&A]
The slug auto turn into number when insert into DB.
It should be
`{"en":"english-1","zh":"中文-2"}`
instead of this
`{"en":"english-1","zh":"-2"}`
Is there anyway to fix this?

**Comment by entity** [comment]
Hey @freekmurze facing this as well,

So for example:
"אמא ואבא"

Generates `<url>/model/-2`

**Reply by freekmurze** [reply]
Feel free to send a PR!
